Cargo.toml

```diff
@@ -14,9 +14,10 @@ keywords = ["cnab"]
 maintenance = { status = "experimental" }
 
 [dependencies]
-semver = { version = "0.9", features = ["serde"] }
-serde = { version = "1.0", features = ["derive"] }
-serde_json = "1.0"
+canonical_json = { git = "https://github.com/engineerd/canonical_json" }
```

Comment:
FYI: If you plan on publishing this on crates.io, you won't be able to publish with a git based dependency.

Reply:
Oh, thanks, didn't know that!
That being said, I personally wouldn't want to merge this in this state, without a canonical JSON library that supports the latest version of Serde.